src/extension.ts

```diff
@@ -53,6 +53,10 @@ function onDidOpen(document) {
     }
 }
 
+function setKeybindingsEnabledContext() {
+    let keybindingsEnabled = vscode.workspace.getConfiguration().get('calva.keybindingsEnabled');
```

Comment:
I try to reduce repeated string literals for ease of refactoring later and reduced chance of errors via typos. I think `calva.keybindingsEnabled` should be in a var, and I'd colocate that var with the related code, but since it's a global setting, I'd put it in `config.ts` next to the repl file extension var here: https://github.com/BetterThanTomorrow/calva/blob/4b8d5c3ee0447e11148465fc09f6fd27123ff1da/src/config.ts#L2, then import it.

The same should be done for the `calva:keybindingsEnabled` context. So they could be like `KEYBINDINGS_ENABLED_CONFIG_KEY` and `KEYBINDINGS_ENABLED_CONTEXT_KEY` or something similar (I'm open to better names if you have ideas).

Reply:
I have made these changes but be aware that the config and context keys are mentioned throughout `package.json` and cannot be changed there.

There is also the command `calva.toggleKeybindingsEnabled`, would you want this in `config.ts` as `TOGGLE_KEYBINDINGS_COMMAND_KEY` as well?

Reply:
No, that's fine since it's only mentioned once anyway. The commands generally can just stay string literals like that for this reason.